packages/common/infra/src/livedata/index.ts

```diff
@@ -170,6 +170,67 @@ export class LiveData<T = unknown> implements InteropObservable<T> {
     );
   }
 
+  /**
+   * @example
+   * ```ts
+   * const a = new LiveData('v1');
+   * const v1 = new LiveData(100);
+   * const v2 = new LiveData(200);
+   *
+   * const v = LiveData.computed(get => {
+   *   return get(a) === 'v1' ? get(v1) : get(v2);
+   * });
+   *
+   * expect(v.value).toBe(100);
+   * ```
+   */
+  static computed<T>(
+    compute: (get: <L>(data: LiveData<L>) => L) => T
+  ): LiveData<T> {
+    return LiveData.from(
+      new Observable(subscribe => {
+        const execute = (next: () => void) => {
+          const subscriptions: Subscription[] = [];
+          const getfn = <L>(data: LiveData<L>) => {
+            let value = null as L;
```

Comment:
init it with data.value?

Reply:
Since livedata.subscribe will definitely output a value immediately, it is the same.